_data/schemaStandard.yaml

```diff
@@ -37,7 +37,9 @@ administrations:
         name:
           en: str()
           fr: str()
-    status: enum('deprecated', 'mandatory', 'preferred')
+    status:
+      en: enum('deprecated', 'mandatory', 'preferred')
+      fr: enum('obsolete','mandatoire','favoris')
```

Comment:
Translations are here https://github.com/canada-ca/ore-ero/blob/ae2eacf96f8bb893bfa0b7cff058d408487efc92/_data/i18n/form.yml#L334-L342

Reply:
So I change mandatory and preferred for maintained and retired ?